python_src/src/lamp_py/performance_manager/l1_rt_metrics.py

```diff
@@ -639,34 +698,31 @@ def process_metrics_table(
             sa.and_(
                 travel_times_cte.c.trip_stop_hash
                 == dwell_times_cte.c.trip_stop_hash,
-                sa.func.coalesce(dwell_times_cte.c.dwell_time_seconds, 0) > 0,
             ),
             full=True,
         )
         .join(
-            headways_cte,
+            headways_branch_cte,
             sa.and_(
                 sa.func.coalesce(
                     travel_times_cte.c.trip_stop_hash,
                     dwell_times_cte.c.trip_stop_hash,
                 )
-                == headways_cte.c.trip_stop_hash,
-                sa.or_(
-                    sa.func.coalesce(headways_cte.c.headway_branch_seconds, 0)
-                    > 0,
-                    sa.func.coalesce(headways_cte.c.headway_trunk_seconds, 0)
-                    > 0,
-                ),
+                == headways_branch_cte.c.trip_stop_hash,
             ),
             full=True,
         )
-        .where(
-            sa.or_(
-                sa.func.coalesce(travel_times_cte.c.travel_time_seconds, 0) > 0,
-                sa.func.coalesce(dwell_times_cte.c.dwell_time_seconds, 0) > 0,
-                sa.func.coalesce(headways_cte.c.headway_branch_seconds, 0) > 0,
-                sa.func.coalesce(headways_cte.c.headway_trunk_seconds, 0) > 0,
```

Comment:
Remove calculation limitations from `all_new_metrics` CTE, just join result sets from `travel_times`, `dwell_times`, and `headways`

Reply:
this is the where clause that was causing the issue?

Reply:
No, this one wasn't causing the specific issue we're fixing. But this join logic becomes unnecessary if the input sets don't have NULL or negative values.